include/deal.II/grid/grid_tools.h

```diff
@@ -1396,7 +1396,7 @@ namespace GridTools
                 // this line has children
                 cell->face(face)->child(0)->vertex(1)
                   = (cell->face(face)->vertex(0) +
-                     cell->face(face)->vertex(1)) / 2;
+                     cell->face(face)->vertex(1)) / 2.;
```

Comment:
> Uh, do you need this now? 

No, this is not needed. This is a leftover from the cleanup - I needed it for an intermediate stage.
I left those changes in because it is good style to use a floating point literal in this case.